src/main/java/seedu/duke/util/DateTimeManager.java

```diff
@@ -1,7 +1,32 @@
 package seedu.duke.util;
 
+import seedu.duke.data.exception.SystemException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 /**
  * Represents a DateTimeManager. Manages the date and time info.
  */
 public class DateTimeManager {
+    public static final String DATEFORMAT = "yyyy-MM-dd HH:mm";
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern(DATEFORMAT);
+
+    /**
+     * Parses a user input into the relevant datetime format specified in the manager.
+
+     * @param input User input to be parsed
+     * @return DateTime representation of the user input
+     * @throws SystemException Occurs when the format of the input is wrong and in the specified format.
+     */
+    public static LocalDateTime dateTimeParser(String input) throws SystemException {
+        LocalDateTime dateTime;
+        try {
+            dateTime = LocalDateTime.parse(input, FORMATTER);
+        } catch (DateTimeParseException e) {
+            throw new SystemException(SystemException.ExceptionType.EXCEPTION_WRONG_TIMING);
+        }
+        return dateTime;
+    }
 }
```

Comment:
This function can be moved into Parser. The original intention of this class is to create a list of acceptable date time formats and returns date time info in a specific format.

Reply:
Alright. I moved them from parser over to here because I thought it would be a central location + parser was getting very bulky already